src/components/page-header/page-header.component.html

```diff
@@ -72,17 +72,29 @@ <h1 class="page-header-title">{{header}}</h1>
 
 </div>
 
-<div *ngIf="secondaryNavigation && ((selectedRoot$ | async) !== (selected$ | async))"
-    class="page-header-secondary" [ngClass]="secondaryNavigationAlignment" role="navigation">
-
-    <ux-tabset *ngIf="(selectedRoot$ | async)?.children; let children" [manual]="true">
-        <ux-tab *ngFor="let child of children"
-            [heading]="child.title"
-            [active]="child === (selected$ | async)"
-            (activated)="select(child)"
-            [uxPageHeaderNavigationSecondaryItem]="child"
-            [disabled]="child.disabled">
-        </ux-tab>
-    </ux-tabset>
+<!-- Secondary Header Section -->
+<div *ngIf="secondaryNavigation && ((selectedRoot$ | async) !== (selected$ | async))" class="ux-page-header-secondary">
 
+    <!-- Secondary Navigation Leading Content -->
+    <div class="page-header-secondary-leading-content" *ngIf="secondaryNavigationLeadingContentTemplate">
```

Comment:
There's an issue with `secondaryNavigationAlignment="center"` when only the trailing template has been applied, it occupies the full width of the secondary nav bar which could cause problems e.g. if a consumer wanted to set the background-color of their custom content.

I fixed this by changing this `ngIf` to `*ngIf="secondaryNavigationLeadingContentTemplate || secondaryNavigationTrailingContentTemplate"`. Having a second empty element fixes the alignment and keeps the trailing element at its natural size (assuming you make the quantum-ux-aspects changes that I have recommended).